README.md

```diff
@@ -2,3 +2,16 @@
 
 a discord bot, written using [discordgo](https://github.com/bwmarrin/discordgo), for ✨ breaking images ✨.
 
+## Running the bot
+
+_`TODO`_
+
+### MacOS
+
+Imagemagick 6 is required, however the version that installs itself if you run `brew install imagemagick@6` doesn't include the `lqr` libary, so using [this](https://github.com/nint8835/homebrew-formulae/blob/main/Formula/imagemagick%406.rb) custom homebrew formula, run:
+
+```sh
+brew install nint8835/formulae/imagemagick@6 # rosettabrew if using apple silicon
```

Comment:
```suggestion
brew install nint8835/formulae/imagemagick@6
```
`rosettabrew` is a custom alias I have to `/usr/local/bin/brew` (as homebrew for ARM is in `/opt/homebrew/bin/brew` which takes priority in my path), so it should probably be omitted here

Reply:
oh i thought it was a actual thing lol